wsproto/frame_protocol.py

```diff
@@ -142,6 +142,28 @@ def _consume_exactly(self, nbytes):
             yield
         return (yield from self._consume_at_most(nbytes))
 
+    def _parse_extended_payload_length(self, opcode, payload_len):
+        if opcode.iscontrol() and payload_len > 125:
+            raise ParseFailed("Control frame with payload len > 125")
+        if payload_len == 126:
+            data = yield from self._consume_exactly(2)
+            (payload_len,) = struct.unpack("!H", data)
+            if payload_len <= 125:
+                raise ParseFailed(
+                    "Payload length used 2 bytes when 1 would have sufficed")
+        elif payload_len == 127:
+            data = yield from self._consume_exactly(8)
+            (payload_len,) = struct.unpack("!Q", data)
+            if payload_len < 2 ** 16:
```

Comment:
Yeah, so I get that. What isn't clear to me is whether this is clearer. I am *not* super familiar with RFC 6455, but this reads to me as being `if payload_len < MAX_16_BIT_UINT`. The exception on the next line basically acts as a code comment explaining what the check is. So, I don't know that the named constant would help much, if at all.